src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -526,6 +527,7 @@ class Shuttle (
             val flightNames = flightPlan.keys.joinToString(", ") { it.name }
             val exceptionLog = "Encountered exception $ex while integrating flight plan containing flight(s) $flightNames"
             writeLog(flightNames, setOf(exceptionLog, ex.stackTrace.toString()), IntegrationStatus.FAILED)
+            if (!isShuttleServer) throw java.lang.Exception(exceptionLog)
```

Comment:
This makes more sense but instead of creating a new exception, can you change to `throw ex`, re-using the existing exception?

Reply:
How about just `            MissionControl.fail(1, flightPlan.keys.first(), ex, listOf(uploadingExecutor))
`

no isShuttleServer?

Reply:
(I added this but that also means other stuff is not necessary anymore?)